arch/mips/boot/dts/img/pistachio_marduk.dts

```diff
@@ -90,6 +90,17 @@
 		reg = <0>;
 		spi-max-frequency = <50000000>;
 	};
```

Comment:
Its missing this commit https://github.com/CreatorDev/openwrt/blob/ci40/target/linux/pistachio/patches-4.1/0097-dts-set-linux-mtd-name-for-flash-devices.patch